AITS/TemperatureAIT.H

```diff
@@ -0,0 +1,50 @@
+#ifndef AITS_TEMPERATURE_H
+#define AITS_TEMPERATURE_H
+
+#include "CouplingDataUser.H"
+#include "fvCFD.H"
+
+namespace preciceAdapter
+{
+namespace AITS
```

Comment:
I am not sure I understand why we need a separate module, instead of implementing the same functionality as a different writer/reader of the CHT module. I think we could save some code duplication.